include/picongpu/param/density.param

```diff
@@ -21,8 +21,30 @@
 /** @file
  *
  * Configure existing or define new normalized density profiles here.
- * During particle species creation in speciesInitialization.param,
+ * During particle species creation in `speciesInitialization.param`,
  * those profiles can be translated to spatial particle distributions.
+ *
+ * This profile is normalized to units of `picongpu::SI::BASE_DENSITY_SI`, also defined in this file.
```

Comment:
Could you please copy the documentation to all `density.pram` files. I know that this will duplicate the comment but we are doing it for `species.param` and other files too.

Reply:
Done